content/post/proficiency_v_creativity.md

```diff
@@ -0,0 +1,63 @@
+---
+author: Caitlin Moorman
+title: "Proficiency v. Creativity"
+cover: ""
+draft: true
+publishDate: "2020-06-01"
+tags:
+ - org
+ - teams
+ - management
+ - creativity
+---
+
+ I recently read [The Culture Code](https://www.indiebound.org/book/9780804176989) by Daniel Coyle, and among the many gems about building great organizational cultures, one idea stood out to me specifically for my own team: managing for proficiency versus managing for creativity. Proficiency-focused organizations must consistently and reliably perform clearly defined tasks, while creativity-focused organizations require consistent innovation and fresh ideas.
+
+Coyle argues that these two types of organizations require substantially different cultures to optimize for either proficiency or creativity. But successful data teams require both - so how do we lead teams that balance the two?
+
+<!--more-->
+
+## Proficiency
+Proficiency is critical for data teams: unreliable systems or inaccurate data destroy the trust placed in us. Without proficiency in data collection, transformation, and reporting, the value of anything built on top can become highly questionable.
+
+Coyle describes that leading high-proficiency groups requires emphasizing clear, simple practices that align behaviors with consistent, reliable outcomes. A lot of [engineering best practices](https://www.locallyoptimistic.com/post/analytics-engineer/) increasingly adopted by data teams are proficiency-focused. Examples of these practices include:
+
+- [Testing and monitoring](https://www.youtube.com/watch?v=M_cNspn2XsE&feature=youtu.be)
+- Automation
+- Version control
+- Code review
+- Runbooks and templates
+- [Creating maintenance plans for new features](https://www.locallyoptimistic.com/post/web-analytics-process/)
+- [Rigorous experimentation](https://www.locallyoptimistic.com/post/ab_dialogue/)
+
+Creating a team culture that values these behaviors (ensuring they don’t become [glue work](https://www.locallyoptimistic.com/post/glue-work/)) increases the reliability of results, and protects the institutional capital of the Analytics team. We’ve covered a lot of tactics here on Locally Optimistic to build the [data quality flywheel](https://www.locallyoptimistic.com/post/data-dies-in-darkness/) and improve the focus on proficiency within the data team (with help from the rest of the company).
```

Comment:
> (ensuring they don’t become [glue work](https://www.locallyoptimistic.com/post/glue-work/))

Do we consider "glue work" to be a bad thing in itself or just the fact that it _is_ important work but often under-appreciated? Maybe we want to say that this sort of work shouldn't go unappreciated (as glue work often does)?